evm/src/trace/decoder.rs

```diff
@@ -225,6 +238,8 @@ impl CallTraceDecoder {
     }
 
     pub fn decode(&self, traces: &mut CallTraceArena) {
```

Comment:
if we need to use async in here can't we make this function async and push the need for a runtime higher up the stack?